tests/framework/ngf.go

```diff
@@ -78,7 +78,7 @@ func InstallNGF(cfg InstallationConfig, extraArgs ...string) ([]byte, error) {
 
 // UpgradeNGF upgrades NGF. CRD upgrades assume the chart is local.
 func UpgradeNGF(cfg InstallationConfig, extraArgs ...string) ([]byte, error) {
-	crdPath := filepath.Join(cfg.ChartPath, "crds")
+	crdPath := filepath.Join(cfg.ChartPath, "crds") + "/"
```

Comment:
```suggestion
	crdPath := filepath.Join(cfg.ChartPath, "crds/")
```

unless `filepath.Join` removes the trailing slash, then your original solution works

Reply:
Yea I had tried that, however it didn't work and when I printed the crdPath variable it wasn't there either. I looked and this line is in the function description: "The result is Cleaned." So I'm assuming it does remove the trailing slash which makes this not as clean.